expressEngine/product/service/product.js

```diff
@@ -2,7 +2,7 @@
 import { BaseService, DBService } from '../../lib/service/index';
 
 // import collections
-import { ProductModel, ProductDetail } from '../../model/index';
+import { Product, ProductDetail } from '../../model/index';
```

Comment:
Don't include index in import statements

Reply:
okay, excluded.